crafting-room-frontend/.env.example

```diff
@@ -0,0 +1,38 @@
+# The host URL for Strapi.
+# - Required!
+STRAPI_HOST=""
```

Comment:
Why the change from URL to HOST? This will mean me doing some manual edits to the current deployment, so I would rather not change it unless necessary.

Reply:
for semantics: URL implies that the provided url is the full one, but in reality, there are several parts that make up a full request URL

hostname + '/api' prefix + endpoint

i myself added the '/api' prefix without realising that was wrong

am aware that this breaks deployment environments though, so I can revert this back.

if you can confirm this again if you want this reverted, ill go ahead and do it 👍